app/forms/provider_interface/cancel_interview_wizard.rb

```diff
@@ -4,7 +4,7 @@ class CancelInterviewWizard
 
     attr_accessor :cancellation_reason
 
-    validates :cancellation_reason, presence: true
+    validates :cancellation_reason, presence: true, length: { maximum: 10240 }
```

Comment:
We seem to be using `word_count` to restrict content across the app. Shouldn't the same apply in this instance?

Reply:
It is specified in this doc as a character count rather than a word count: https://docs.google.com/document/d/1r7ANbveFFcgkRyiiyvCqlpfufNRz5UdHLL17n6FDx0E

I've tweaked the number to be consistent with other free text fields in the API, but I'm open to changing it if there's a reason to. Let's bring this up when we review the feature